frontend/src/utils/TestCommonWrapper.tsx

```diff
@@ -0,0 +1,47 @@
+import { useKeycloak } from '@react-keycloak/web';
+import { IRole } from 'interfaces';
+import { IAgency } from 'interfaces/agency';
+import React from 'react';
+import { TenantProvider } from 'tenants';
+
+import TestProviderWrapper from './TestProviderWrapper';
+import TestRouterWrapper from './TestRouterWrapper';
+
+jest.mock('@react-keycloak/web');
```

Comment:
Wonder if there is a way to also remind the test to clean this up at the end.

Reply:
I'm still hoping we turn on the jest flag to reset mock states between tests.